### PR TITLE
content updates

### PR DESCRIPTION
**Pull request by J05h-L**
### Context
First set of content updates for school pages

### Checklist
- [x] Work items have been linked [(use AB#)](https://learn.microsoft.com/en-us/azure/devops/boards/github/link-to-from-github?view=azure-devops#use-ab-to-link-from-github-to-azure-boards-work-items)
- [x] Your code builds clean without any errors or warnings
- [x] You have run all unit/integration tests and they pass
- [x] Your branch has been rebased onto main
- [x] You have tested by running locally

### CODE REVIEW

File: web/src/Web.App/Views/SchoolPlanningCreate/Start.cshtml [hunk 1]
```diff
@@ -18,10 +18,10 @@
             Completing a deployment plan will take about 10 minutes.
         </p>
         <p class="govuk-body">
-            A full list of the data will be asked to input 
+            A full list of the data you will be asked to input
```
Comment (WolfyUK) [inline web/src/Web.App/Views/SchoolPlanningCreate/Start.cshtml:21]: I've fixed this in my other PR, so you'll probably need to revert this change

File: web/src/Web.App/Views/SchoolSpending/Index.cshtml [hunk 2]
```diff
@@ -38,12 +38,22 @@
 
     @foreach (var category in Model.PriorityCosts)
     {
+        var categoryToLower = @category.Rating.Category == "Other costs"
```
Comment (WolfyUK) [inline web/src/Web.App/Views/SchoolSpending/Index.cshtml:41]: There are constants in `Web.App.Domain.Category` that could be used here instead

File: web/src/Web.App/Views/SchoolPlanningCreate/Start.cshtml (hunk 1 above)
Comment (J05h-L) [reply web/src/Web.App/Views/SchoolPlanningCreate/Start.cshtml]: reverted

File: web/src/Web.App/Views/SchoolSpending/Index.cshtml (hunk 2 above)
Comment (J05h-L) [reply web/src/Web.App/Views/SchoolSpending/Index.cshtml]: thanks, fixed in latest commit

**Review by WolfyUK — APPROVED** [review]
